examples/onnx/pytorch/translation/run_onnx_exporter.py

```
@@ -178,39 +173,33 @@ def main():
     logger.setLevel(logging.ERROR)
     transformers.utils.logging.set_verbosity_error()
 
-    if args.model_name_or_path:
-        model, tokenizer = load_model_tokenizer(args.model_name_or_path, local_device)
-    else:
-        raise ValueError("Make sure that model name has been passed")
-
-    if model.config.decoder_start_token_id is None:
-        raise ValueError("Make sure that `config.decoder_start_token_id` is correctly defined")
-
+    device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
```

Is it actually faster to use CUDA for this? If not, just default to CPU.
Also this logic makes the default value of the device arg misleading. It says it defaults to "cpu" but the actual behavior depends on `torch.cuda.is_available`. If we can always default to CPU, then just replace this with: `device = torch.device(args.device)`

---

Forgot that I've set the default value to CPU in the beginning, so remove this line so that the default value is CPU, matter what value torch.cuda.is_available() is.